scheduler.go

```diff
@@ -306,23 +365,6 @@ func (s *Scheduler) Do(jobFun interface{}, params ...interface{}) (*Job, error)
 		}
 	}
 
-	if !j.startsImmediately {
-
-		if j.lastRun == s.time.Unix(0, 0) {
```

Comment:
this was a workaround that had impact on the job scheduling logic and go removed